lib/ohloh_scm/git/validation.rb

```diff
@@ -3,10 +3,16 @@
 module OhlohScm
   module Git
     class Validation < OhlohScm::Validation
+      def validate
+        super
+        @errors << [:branch_name, 'Invalid Branch Name.'] if scm.branch_name.to_s.empty?
+      end
+
       private
 
       def validate_server_connection
-        msg = "The server did not respond to the 'git-ls-remote' command. Is the URL correct?"
+        msg = "The server did not respond to the 'git-ls-remote' command."
+        msg << ' Is the URL and Branch correct?'
```

Comment:
Can we please change this to ' Are the URL and Branch fields correct?'

Reply:
Yes, its updated!